jruby-gradle-base-plugin/src/main/groovy/com/github/jrubygradle/GemUtils.groovy

```diff
@@ -97,6 +97,7 @@ class GemUtils {
                 // should already have taken care of, see #79
                 environment 'JBUNDLE_SKIP', true
                 environment 'JARS_SKIP', true
+                environment 'GEM_HOME', destDir.absolutePath
```

Comment:
if you set GEM_HOME to prevent the install to use some outer ruby environment then you should also set GEM_PATH as rubygems looks there if the gem is already installed and if not found on GEM_PATH it will install it on GEM_HOME